.github/workflows/unit-tests.yml

```
@@ -45,21 +45,13 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - php: '8.0'
-            wordpress: '6.2.2'
+          - php: '8.2'
+            wordpress: '6.3.1'
```

We don't need to pin to a minor release:

```suggestion
            wordpress: '6.3'
```